### PR TITLE
Multiple ImperiHome binding fixes

### PR DESCRIPTION
**Pull request by pdegeus**
Multiple ImperiHome binding fixes:
- Fixes NPE for 'trippable' devices (motion, contact) when state is null
- Added option to configure system ID
- Added support for custom 'defaultIcon' in ImperiHome

### CODE REVIEW

File: addons/io/org.openhab.io.imperihome/README.md [hunk 1]
```diff
@@ -7,12 +7,40 @@ It creates a REST service at _/imperihome/iss_ that implements the [ImperiHome S
 
 The ImperiHome integration service can be installed through the Paper UI. Navigate to Extensions &gt; Misc and click Install.
 
+If you want to use the latest snapshot or your own build of this binding, compile the code and copy the resulting JAR file into the _openhab/addons_ directory.
```
Comment (kaikreuzer) [inline addons/io/org.openhab.io.imperihome/README.md]: This is nothing Imperi-Home add-on specific, so I would suggest to remove this here.

File: addons/io/org.openhab.io.imperihome/README.md [hunk 2]
```diff
@@ -7,12 +7,40 @@ It creates a REST service at _/imperihome/iss_ that implements the [ImperiHome S
 
 The ImperiHome integration service can be installed through the Paper UI. Navigate to Extensions &gt; Misc and click Install.
```
Comment (kaikreuzer) [inline addons/io/org.openhab.io.imperihome/README.md]: Just noticed: "Extensions" is by now called "Add-ons", so you might want to update this.

File: addons/io/org.openhab.io.imperihome/README.md [hunk 3]
```diff
@@ -7,12 +7,40 @@ It creates a REST service at _/imperihome/iss_ that implements the [ImperiHome S
...
 
+If you want to use the latest snapshot or your own build of this binding, compile the code and copy the resulting JAR file into the _openhab/addons_ directory.
+
+<a name="configuration"></a>
+
 ## Configuration
 
-The service itself has no configuration. ImperiHome on the other hand must be configured to connect to your openHAB instance.
+### openHAB binding
```
Comment (kaikreuzer) [inline addons/io/org.openhab.io.imperihome/README.md]: change to "Add-on" or "Integration" - this is actually NOT a binding.

File: addons/io/org.openhab.io.imperihome/README.md [hunk 4]
```diff
@@ -7,12 +7,40 @@ It creates a REST service at _/imperihome/iss_ that implements the [ImperiHome S
...
+
+To configure the ImperiHome integration binding in openHAB, create a _imperihome.cfg_ file in the _conf/services_ directory. The following configuration options can be used:
```
Comment (kaikreuzer) [inline addons/io/org.openhab.io.imperihome/README.md]: replace "binding"

File: addons/io/org.openhab.io.imperihome/src/main/java/org/openhab/io/imperihome/internal/ImperiHomeConfig.java [hunk 5]
```diff
@@ -0,0 +1,48 @@
+package org.openhab.io.imperihome.internal;
```
Comment (kaikreuzer) [inline addons/io/org.openhab.io.imperihome/src/main/java/org/openhab/io/imperihome/internal/ImperiHomeConfig.java]: license header missing

**Comment by pdegeus** [comment]
@kaikreuzer Thanks for the review :) I've pushed the suggested changes.

**Pull request merged**